e2e/src/pay.spec.ts

```diff
@@ -6,21 +6,18 @@ import { gql, GraphQLClient } from "graphql-request";
 import type { SessionData } from "@opensystemslab/planx-core/types";
 import type { Context } from "./context";
 import {
+  contextDefaults,
   getGraphQLClient,
   setUpTestContext,
   tearDownTestContext,
 } from "./context";
 
 let context: Context = {
-  user: {
-    firstName: "test",
-    lastName: "test",
-    email: "e2epaytest@test.com",
-  },
+  ...contextDefaults,
   team: {
     name: "Buckinghamshire",
     slug: "buckinghamshire",
-    logo: "https://placedog.net/250/250",
+    logo: "https://raw.githubusercontent.com/theopensystemslab/planx-team-logos/main/planx-testing.svg",
     primaryColor: "#F30415",
     homepage: "example.com",
   },
```

Comment:
Are we relying on Bucks to pass the `isSupported` check in `api.planx.uk/pay/pay.ts` and for a valid API key?

If we could have all tests running again an independent team it could help with DX not always having to destroy / recreate test containers if all content was encapsulated within a single team.

Reply:
Exactly, a test team with credentials would be a nice improvement here.